tests/v2/validation/projects/projects.go

```diff
@@ -47,37 +94,86 @@ var prtb = v3.ProjectRoleTemplateBinding{
 	UserPrincipalName: "",
 }
 
-func createProject(client *rancher.Client, clusterName string) (*v3.Project, error) {
+func createProject(client *rancher.Client, clusterID string, namespacePodLimit, projectPodLimit, cpuLimit, cpuReservation, memoryLimit, memoryReservation string) (*v3.Project, error) {
```

Comment:
What are your thoughts on this? can we create a project template similar to https://github.com/rancher/shepherd/blob/main/extensions/workloads/template.go#L27 and add then pass that object to create a project like you did in line 150 in this file. is that possible?

Reply:
I'm not entirely convinced that using a template provides significant benefits in this context, unless there's something I'm overlooking.

Reply:
I agree, good point! Will look into this